scripts/prepare_sector_network.py

```diff
@@ -2391,14 +2391,20 @@ def add_EVs(
         )
 
         # Add vehicle-to-grid if enabled
-        if options["v2g"]:
+        if options["v2g"] or options["v2g"] in (True, 1):
```

Comment:
The condition is not optimal but works, we can have sth like:
```
if options ["v2g"]:
      v2g_share = 1.0 if options["v2g"] is True else float(options["v2g"])
```

Reply:
Indeed! But I tested it, and you can savely convert `bool` to `float`, no problem :+1: